astrago-airgap/deploy-kubespray.sh

```
@@ -0,0 +1,1174 @@
+        exit 1
+    fi
+
+    echo -e "${GREEN}✓ roles/offline-repo 존재${NC}"
+    echo ""
+}
+
+# ansible 설치 확인 및 설정
+check_or_setup_ansible() {
+    local outputs_dir="${KUBESPRAY_OFFLINE_DIR}/outputs-${VERSION}"
+    local kubespray_dir="${outputs_dir}/kubespray-${VERSION}"
+    local requirements_txt="${kubespray_dir}/requirements.txt"
```

Shellcheck (0.11.0)

[warning] requirements_txt appears unused. Verify use (or export if used externally).

(SC2034)